sqlest/src/test/scala/sqlest/sql/SelectStatementBuilderSpec.scala

```diff
@@ -549,6 +549,22 @@ class SelectStatementBuilderSpec extends BaseStatementBuilderSpec {
     )
   }
 
+  def testTransform[A, R <: Relation](query: Select[A, R]) =
+    query
+      .innerJoin(TableTwo).on(TableOne.col2 === TableTwo.col2)
+      .where(TableOne.col2 =!= "SomeString".constant)
+
+  "query with a transform addition" should "produce the right sql" in {
+    sql {
+      select(MyTable.col1, MyTable.col2.?)
+        .from(MyTable)
+        .transform(testTransform _)
+    } should equal(
+      """select mytable.col1 as mytable_col1, mytable.col2 as mytable_col2 from mytable inner join two on (one.col2 = two.col2) where (one.col2 <> 'SomeString')""",
```

Comment:
You can use multiline strings to make this SQL a bit easier to read in the test (see the test just above):
```
s"""
|select mytable.col1 as mytable_col1,
|etc.
""".formatSql